### PR TITLE
CTOR - 815 : add(network::cisco::standard::snmp): add option --add-fc-fe-errors

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Monitor OID fcIfTxWaitCount (.1.3.6.1.4.1.9.9.289.1.2.1.1.15)

CTOR-815

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 1]
```diff
@@ -32,21 +30,21 @@ Le connecteur apporte les modèles de service suivants
...
-| Spanning-Tree | Net-Cisco-Standard-SpanningTree-SNMP-custom  | Contrôle l'état du spanning tree sur les interfaces    |            |
-| Stack         | Net-Cisco-Standard-Stack-SNMP-custom         | Contrôle l'état de la "stack" Cisco                    |            |
-| Voice-Call    | Net-Cisco-Standard-Voice-Call-SNMP-custom    | Contrôle les statistiques d'appels                     |            |
+| Alias         | Modèle de service                            | Description                                                   | Découverte |
+|:--------------|:---------------------------------------------|:--------------------------------------------------------------|:----------:|
+| Aaa-Servers   | Net-Cisco-Standard-Aaa-Servers-SNMP-custom   | Contrôle les serveurs AAA                                     | X          |
+| Anycast       | Net-Cisco-Standard-Anycast-SNMP-custom       | Contrôle le type de trafic                                    |            |
+| Arp           | Net-Cisco-Standard-Arp-SNMP-custom           | Contrôle la table arp                                         |            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:37]:
```suggestion
| Arp           | Net-Cisco-Standard-Arp-SNMP-custom           | Contrôle la table ARP                                         |            |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 2]
```diff
@@ -59,18 +57,18 @@ Le connecteur apporte les modèles de service suivants
...
+#### Découverte de services
 
-| Nom de la règle                         | Description                                          |
-|:----------------------------------------|:-----------------------------------------------------|
-| Net-Cisco-Standard-SNMP-Aaa-Server-Name | Découvre les serveurs AAA et supervise l'utilisation |
-| Net-Cisco-Standard-SNMP-Interface-Name  | Découvre les interfaces et supervise l'utilisation   |
+| Nom de la règle                         | Description                                                             |
+|:----------------------------------------|:------------------------------------------------------------------------|
+| Net-Cisco-Standard-SNMP-Aaa-Server-Name | Découvre les serveurs AAA et supervise l'utilisation                                                                        |
+| Net-Cisco-Standard-SNMP-Interface-Name  | Découvre les interfaces réseaux et supervise le statut et l'utilisation |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:71]:
```suggestion
| Net-Cisco-Standard-SNMP-Interface-Name  | Découvre les interfaces réseau et supervise leur statut et leur utilisation |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 3]
```diff
@@ -420,291 +412,305 @@ yum install centreon-plugin-Network-Cisco-Standard-Snmp
...
-| CRITICALTOTAL                    | Thresholds                                                                                                                                              |                      |             |
-| EXTRAOPTIONS                     | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                     | --verbose            |             |
+| Macro                            | Description                                                                                                                                      | Valeur par défaut    | Obligatoire |
+|:---------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------|:-----------:|
+| FILTERNAME                       | Filter AAA server by name (E.g.: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\])                                         |                      |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:417]:
```suggestion
| FILTERNAME                       | Filter AAA server by name (Example: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\])                                         |                      |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 4]
```diff
@@ -420,291 +412,305 @@ yum install centreon-plugin-Network-Cisco-Standard-Snmp
...
 
 </TabItem>
 <TabItem value="Hsrp" label="Hsrp">
 
 | Macro        | Description                                                                                                                                      | Valeur par défaut | Obligatoire |
 |:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
-| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states. (Default: 'primary') | primary           | X           |
-| EXTRAOPTIONS | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                              | --verbose         |             |
+| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. Ifrole is 'secondary', an error if HSRPs are 'active' states.                      | primary           | X           |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:536]:
```suggestion
| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states.                      | primary           | X           |
```

Comment (cg-tw) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:536]: the sentences make no sense

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 5]
```diff
@@ -420,291 +412,305 @@ yum install centreon-plugin-Network-Cisco-Standard-Snmp
...
-| EXTRAOPTIONS            | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                                                                                | --verbose --use-new-perfdata                          |             |
+| Macro                   | Description                                                                                                                                                              | Valeur par défaut                                     | Obligatoire |
+|:------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------|:-----------:|
+| UNITSTRAFFIC            | Units of thresholds for the traffic ('percent\_delta', 'bps', 'counter')                                                                                                 | percent\_delta                                        |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:544]:
```suggestion
| UNITSTRAFFIC            | Units in which the thresholds are given for the traffic ('percent\_delta', 'bps', 'counter')                                                                                                 | percent\_delta                                        |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 6]
```diff
@@ -867,77 +873,77 @@ Les options disponibles pour chaque modèle de services sont listées ci-dessous
...
+| --statefile-cipher       | Define the cipher algorithm to encrypt the cache (default: 'AES').                                                                                                                                                                            |
 | --filter-counters        | Only display some counters (regexp can be used). Example: --filter-counters='auth'                                                                                                                                                            |
-| --filter-name            | Filter AAA server by name (e.g.: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\]).                                                                                                                                     |
+| --filter-name            | Filter AAA server by name (E.g.: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\]).                                                                                                                                     |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:885]:
```suggestion
| --filter-name            | Filter AAA server by name (Example: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\]).                                                                                                                                     |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 7]
```diff
@@ -867,77 +873,77 @@ Les options disponibles pour chaque modèle de services sont listées ci-dessous
...
+| --show-cache                                    | Display cache interface data.                                                                                                                                                                                                                                                                                                                                               |
 
 </TabItem>
 <TabItem value="Arp" label="Arp">
 
 | Option                   | Description                                                                      |
 |:-------------------------|:---------------------------------------------------------------------------------|
 | --filter-macaddr         | Filter mac addresses (can be a regexp).                                          |
-| --filter-ipaddr          | Filter IP addresses (can be a regexp).                                           |
+| --filter-ipaddr          | Filter ip addresses (can be a regexp).                                           |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:946]:
```suggestion
| --filter-ipaddr          | Filter IP addresses (can be a regexp).                                           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 8]
```diff
@@ -1002,55 +1008,56 @@ Les options disponibles pour chaque modèle de services sont listées ci-dessous
 | Option        | Description                                                                                                                                         |
 |:--------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
 | --filter-vrid | Filter VRID (can be a regexp).                                                                                                                      |
-| --role        | If role is 'primary', an error if HSRPs are 'standby' states. Ifrole is 'secondary', an error if HSRPs are 'active' states. (Default: 'primary')    |
+| --role        | If role is 'primary', an error if HSRPs are 'standby' states. Ifrole is 'secondary', an error if HSRPs are 'active' states. (default: 'primary')    |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:1011]:
```suggestion
| --role        | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states. (default: 'primary')    |
```

Comment (cg-tw) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:1011]: same here, doesn't make sense

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 9]
```diff
@@ -1082,16 +1089,16 @@ Les options disponibles pour chaque modèle de services sont listées ci-dessous
 | --warning-usage  | Warning threshold in percent.                                                                                                                                                              |
 | --critical-usage | Critical threshold in percent.                                                                                                                                                             |
 | --filter-pool    | Filter pool to check (can use regexp).                                                                                                                                                     |
-| --check-order    | Check memory in standard cisco mib. If you have some issue (wrong memory information in a specific mib), you can change the order (Default: 'enhanced\_pool,pool,process,system\_ext').    |
+| --check-order    | Check memory in standard cisco mib. If you have some issue (wrong memory information in a specific mib), you can change the order (default: 'enhanced\_pool,pool,process,system\_ext').    |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:1092]:
```suggestion
| --check-order    | Check memory in standard cisco MIB. If you have some issue (wrong memory information in a specific MIB), you can change the order (default: 'enhanced\_pool,pool,process,system\_ext').    |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 10]
```diff
@@ -1127,18 +1134,18 @@ Les options disponibles pour chaque modèle de services sont listées ci-dessous
...
-| --critical-status        | Set critical threshold for member status (Default: '%{state} !~ /ready/ && %{state} !~ /provisioned/'). You can use the following variables: %{name}, %{role}, %{state}  Role can be: 'master', 'member', 'notMember', 'standby'.  State can be: 'waiting', 'progressing', 'added', 'ready', 'sdmMismatch', 'verMismatch', 'featureMismatch', 'newMasterInit', 'provisioned', 'invalid', 'removed'.    |
+| --warning-* --critical-* | Set thresholds for members count for each states.can be: 'waiting', 'progressing', 'added', 'ready', 'sdm-mismatch', 'version-mismatch', 'feature-mismatch', 'new-master-init', 'provisioned', 'invalid', 'removed')                                                                                                                                                                                 |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:1144]:
```suggestion
| --warning-* --critical-* | Set thresholds on members count for each states. (Can be: 'waiting', 'progressing', 'added', 'ready', 'sdm-mismatch', 'version-mismatch', 'feature-mismatch', 'new-master-init', 'provisioned', 'invalid', 'removed')                                                                                                                                                                                 |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 11]
```diff
@@ -22,7 +20,7 @@ The connector brings the following service templates (sorted by the host templat
 
 | Service Alias | Service Template                           | Service Description                                                                                           |
 |:--------------|:-------------------------------------------|:--------------------------------------------------------------------------------------------------------------|
-| Cpu           | Net-Cisco-Standard-Cpu-SNMP-custom         | Check the rate of utilization of the CPU for the machine. This check can give the average utilization rate of the CPU |
+| Cpu           | Net-Cisco-Standard-Cpu-SNMP-custom         | Check the rate of utilization of CPU for the machine. This check can give the average utilization rate of CPU |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:23]:
```suggestion
| Cpu           | Net-Cisco-Standard-Cpu-SNMP-custom         | Check the rate of utilization of the CPU for the machine. This check can give the average utilization rate of the CPU |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 12]
```diff
@@ -31,21 +29,21 @@ The connector brings the following service templates (sorted by the host templat
...
-| Spanning-Tree | Net-Cisco-Standard-SpanningTree-SNMP-custom  | Check Spanning Tree state on interfaces       |            |
-| Stack         | Net-Cisco-Standard-Stack-SNMP-custom         | Check Cisco "stack" state                     |            |
-| Voice-Call    | Net-Cisco-Standard-Voice-Call-SNMP-custom    | Check call traffic statistics                 |            |
+| Service Alias | Service Template                             | Service Description                                                    | Discovery  |
+|:--------------|:---------------------------------------------|:-----------------------------------------------------------------------|:----------:|
+| Aaa-Servers   | Net-Cisco-Standard-Aaa-Servers-SNMP-custom   | Check AAA servers                                                      | X          |
+| Anycast       | Net-Cisco-Standard-Anycast-SNMP-custom       | Check traffic types                                                    |            |
+| Arp           | Net-Cisco-Standard-Arp-SNMP-custom           | Check arp table.                                                       |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:36]:
```suggestion
| Arp           | Net-Cisco-Standard-Arp-SNMP-custom           | Check ARP table.                                                       |            |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 13]
```diff
@@ -422,303 +413,310 @@ yum install centreon-plugin-Network-Cisco-Standard-Snmp
...
-| CRITICALTOTAL                    | Thresholds                                                                                                                                              |                      |             |
-| EXTRAOPTIONS                     | Any extra option you may want to add to the command (e.g. a --verbose flag). All options are listed [here](#available-options)                                                     | --verbose            |             |
+| Macro                            | Description                                                                                                                             | Default value        | Mandatory   |
+|:---------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------|:---------------------|:-----------:|
+| FILTERNAME                       | Filter AAA server by name (E.g.: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\])                                |                      |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:418]:
```suggestion
| FILTERNAME                       | Filter AAA server by name (Example: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\])                                |                      |             |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 14]
```diff
@@ -422,303 +413,310 @@ yum install centreon-plugin-Network-Cisco-Standard-Snmp
...
-|:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
-| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states. (Default: 'primary') | primary           | X           |
-| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options)                                              | --verbose         |             |
+| Macro        | Description                                                                                                                            | Default value     | Mandatory   |
+|:-------------|:---------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. Ifrole is 'secondary', an error if HSRPs are 'active' states.            | primary           | X           |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:537]:
```suggestion
| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states.            | primary           | X           |
```

Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:537]: makes no sense

File: pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 15]
```diff
@@ -874,69 +872,69 @@ All available options for each service template are listed below:
...
+| --statefile-cipher       | Define the cipher algorithm to encrypt the cache (default: 'AES').                                                                                                                                                                            |
 | --filter-counters        | Only display some counters (regexp can be used). Example: --filter-counters='auth'                                                                                                                                                            |
-| --filter-name            | Filter AAA server by name (e.g.: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\]).                                                                                                                                     |
+| --filter-name            | Filter AAA server by name (E.g.: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\]).                                                                                                                                     |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:884]:
```suggestion
| --filter-name            | Filter AAA server by name (Example: 10.199.126.100:1812:1813. Format: \[address\]:\[authPort\]:\[accPort\]).                                                                                                                                     |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 16]
```diff
@@ -956,48 +954,48 @@ All available options for each service template are listed below:
...
+| --critical-status | Define the conditions to match for the status to be CRITICAL (default: '%{running\_last\_changed} \> %{running\_last\_saved}'). You can use the following variables: %{running\_last\_changed}, %{running\_last\_saved}, %{startup\_last\_changed}    |
 
 </TabItem>
 <TabItem value="Cpu" label="Cpu">
 
 | Option                   | Description                                                                                                                                                              |
 |:-------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| --check-order            | Check cpu in standard cisco mib. If you have some issue (wrong cpu information in a specific mib), you can change the order (Default: 'process,old\_sys,system\_ext').   |
+| --check-order            | Check CPU in standard cisco mib. If you have some issue (wrong CPU information in a specific mib), you can change the order (default: 'process,old\_sys,system\_ext').   |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:985]:
```suggestion
| --check-order            | Check CPU in standard cisco MIB. If you have some issue (wrong CPU information in a specific MIB), you can change the order (default: 'process,old\_sys,system\_ext').   |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 17]
```diff
@@ -1009,55 +1007,56 @@ All available options for each service template are listed below:
 | Option        | Description                                                                                                                                         |
 |:--------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
 | --filter-vrid | Filter VRID (can be a regexp).                                                                                                                      |
-| --role        | If role is 'primary', an error if HSRPs are 'standby' states. Ifrole is 'secondary', an error if HSRPs are 'active' states. (Default: 'primary')    |
+| --role        | If role is 'primary', an error if HSRPs are 'standby' states. Ifrole is 'secondary', an error if HSRPs are 'active' states. (default: 'primary')    |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:1010]:
```suggestion
| --role        | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states. (default: 'primary')    |
```

Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:1010]: same here, makes no sense

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 18]
```diff
@@ -420,291 +412,305 @@ yum install centreon-plugin-Network-Cisco-Standard-Snmp
...
 
 </TabItem>
 <TabItem value="Hsrp" label="Hsrp">
 
 | Macro        | Description                                                                                                                                      | Valeur par défaut | Obligatoire |
 |:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
-| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states. (Default: 'primary') | primary           | X           |
-| EXTRAOPTIONS | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                              | --verbose         |             |
+| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states.                      | primary           | X           |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:536]:
```suggestion
| ROLE         | If role is 'primary', an error is return if HSRPs are 'standby' states. If role is 'secondary', an error is returb if HSRPs are 'active' states.                      | primary           | X           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 19]
```diff
@@ -1002,55 +1008,56 @@ Les options disponibles pour chaque modèle de services sont listées ci-dessous
 | Option        | Description                                                                                                                                         |
 |:--------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
 | --filter-vrid | Filter VRID (can be a regexp).                                                                                                                      |
-| --role        | If role is 'primary', an error if HSRPs are 'standby' states. Ifrole is 'secondary', an error if HSRPs are 'active' states. (Default: 'primary')    |
+| --role        | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states. (default: 'primary')    |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:1011]:
```suggestion
| --role        | If role is 'primary', an error is return if HSRPs are 'standby' states. If role is 'secondary', an error is returb if HSRPs are 'active' states. (default: 'primary')    |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 20]
```diff
@@ -422,303 +413,310 @@ yum install centreon-plugin-Network-Cisco-Standard-Snmp
...
-|:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
-| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states. (Default: 'primary') | primary           | X           |
-| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options)                                              | --verbose         |             |
+| Macro        | Description                                                                                                                            | Default value     | Mandatory   |
+|:-------------|:---------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states.            | primary           | X           |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:537]:
```suggestion
| ROLE         | If role is 'primary', an error is return if HSRPs are 'standby' states. If role is 'secondary', an error is returb if HSRPs are 'active' states.           | primary           | X           |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 21]
```diff
@@ -1009,55 +1007,56 @@ All available options for each service template are listed below:
 | Option        | Description                                                                                                                                         |
 |:--------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
 | --filter-vrid | Filter VRID (can be a regexp).                                                                                                                      |
-| --role        | If role is 'primary', an error if HSRPs are 'standby' states. Ifrole is 'secondary', an error if HSRPs are 'active' states. (Default: 'primary')    |
+| --role        | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states. (default: 'primary')    |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:1010]:
```suggestion
| --role        | If role is 'primary', an error is return if HSRPs are 'standby' states. If role is 'secondary', an error is returb if HSRPs are 'active' states. (default: 'primary')    |
```

**Review by omercier — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 22]
```diff
@@ -420,291 +412,305 @@ yum install centreon-plugin-Network-Cisco-Standard-Snmp
...
 
 </TabItem>
 <TabItem value="Hsrp" label="Hsrp">
 
 | Macro        | Description                                                                                                                                      | Valeur par défaut | Obligatoire |
 |:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
-| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states. (Default: 'primary') | primary           | X           |
-| EXTRAOPTIONS | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                              | --verbose         |             |
+| ROLE         | If role is 'primary', an error is returned when HSRPs are 'standby' states. If role is 'secondary', an error is returned when HSRPs are 'active' states.                      | primary           | X           |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:536]:
```suggestion
| ROLE         | If role is 'primary', an error is returned when HSRPs are in a 'standby' state. If role is 'secondary', an error is returned when HSRPs are in an 'active' state.                      | primary           | X           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 23]
```diff
@@ -1002,55 +1008,56 @@ Les options disponibles pour chaque modèle de services sont listées ci-dessous
 | Option        | Description                                                                                                                                         |
 |:--------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
 | --filter-vrid | Filter VRID (can be a regexp).                                                                                                                      |
-| --role        | If role is 'primary', an error if HSRPs are 'standby' states. Ifrole is 'secondary', an error if HSRPs are 'active' states. (Default: 'primary')    |
+| --role        | If role is 'primary', an error is returned when HSRPs are 'standby' states. If role is 'secondary', an error is returned when HSRPs are 'active' states. Default: 'primary'.    |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:1011]:
```suggestion
| --role        | If role is 'primary', an error is returned when HSRPs are in a 'standby' state. If role is 'secondary', an error is returned when HSRPs are in an 'active' state. Default: 'primary'.    |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md (hunk 22 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:536]:
```suggestion
| ROLE         | If role is 'primary', an error is returned when HSRPs are in a 'standby' states. If role is 'secondary', an error is returned when HSRPs are in a 'active' states.                      | primary           | X           |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 24]
```diff
@@ -422,303 +413,310 @@ yum install centreon-plugin-Network-Cisco-Standard-Snmp
...
-| ROLE         | If role is 'primary', an error if HSRPs are 'standby' states. If role is 'secondary', an error if HSRPs are 'active' states. (Default: 'primary') | primary           | X           |
-| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options)                                              | --verbose         |             |
+| Macro        | Description                                                                                                                            | Default value     | Mandatory   |
+|:-------------|:---------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| ROLE         | If role is 'primary', an error is returned when HSRPs are 'standby' states. If role is 'secondary', an error is returned when HSRPs are 'active' states.           | primary           | X           |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:537]:
```suggestion
| ROLE         | If role is 'primary', an error is returned when HSRPs are in a 'standby' state. If role is 'secondary', an error is returned when HSRPs are in an 'active' state.           | primary           | X           |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md [hunk 25]
```diff
@@ -1009,55 +1007,56 @@ All available options for each service template are listed below:
 | Option        | Description                                                                                                                                         |
 |:--------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
 | --filter-vrid | Filter VRID (can be a regexp).                                                                                                                      |
-| --role        | If role is 'primary', an error if HSRPs are 'standby' states. Ifrole is 'secondary', an error if HSRPs are 'active' states. (Default: 'primary')    |
+| --role        | If role is 'primary', an error is returned when HSRPs are 'standby' states. If role is 'secondary', an error is returned when HSRPs are 'active' states. Default: 'primary'.    |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md:1010]:
```suggestion
| --role        | If role is 'primary', an error is returned when HSRPs are in a 'standby' state. If role is 'secondary', an error is returned when HSRPs are in an 'active' state. Default: 'primary'.    |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**